python/lsst/validate/drp/base.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+
+# LSST Data Management System
+# Copyright 2008-2016 AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <https://www.lsstcorp.org/LegalNotices/>.
+
+from __future__ import print_function, division
+
+
+class ValidateError(Exception):
+    """Base classes for exceptions in validate_drp."""
+    pass
```

Comment:
Hmm.  We should raise this as a more general stylistic issue, maybe on c.l.o., then.  Having a new hierarchy in each module does have its advantages, especially if you're willing to commit to defining subclasses with a lot of multiple inheritance (so you can also inherit from the right built-in exceptions), but it's definitely not standard practice in the stack right now.